Sources/IO/Misc/PDBReader/index.js

```diff
@@ -238,7 +190,6 @@ export function extend(publicAPI, model, initialValues = {}) {
     'xSpacing',
```

Comment:
Ok I thought they were coming from the copy paste of the elevation reader. 
To me it looks like scaling. Is it useful to have asymmetric scaling across axis?